marimo/_messaging/ops.py

```diff
@@ -339,7 +342,12 @@ class KernelCapabilities:
     terminal: bool = False
 
     def __post_init__(self) -> None:
-        self.sql = DependencyManager.has_duckdb()
+        if DependencyManager.has_duckdb():
```

Comment:
I wonder if we could pass the version as an argument to this call. It could provide a better warning too or suggestion to upgrade.